app/models/settings/community.rb

```diff
@@ -10,7 +10,7 @@ class Community < Base
     setting(
       :community_name,
       type: :string,
-      default: ApplicationConfig["COMMUNITY_NAME"] || "New Forem",
+      default: ApplicationConfig["COMMUNITY_NAME"],
```

Comment:
We remove "New Forem", because when a creator signs up and then goes through their onboarding, we want to show them a blank field or what was set by their `ApplicationConfig["COMMUNITY_NAME"]`.

"New Forem" no longer makes sense with this onboarding step at the start.